docs/asyncapi.yaml

```diff
@@ -425,36 +387,9 @@ components:
       type: number
       format: decimal
       description: Decimal net value of request before enrichment
-      minimum: 0.00
+      minimum: 0
```

Comment:
Why remove the decimals here?

Reply:
I didn't.  I removed the duplicate, but guess it didn't have decimals.  Added them